### PR TITLE
Presentation proposal - week3

### PR DESCRIPTION
**Pull request by ChangGabriel**
Presentation proposal.

Is week 3 from 05 april - 09 april or 12 april-16 april?

### CODE REVIEW

**Comment by khesoem** [comment]
Hi,
I think your proposal is in scope of Devops. Can you just make your outline and topic description a bit more detailed?

File: contributions/presentation/week3/gchang-wska/README.md [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+
+William Skagerström (wska@kth.se)
+Github username: wska
+
+## Topic
+An introduction to code coverage
+
+## Outline
+* What is code coverage
+* How it is used in devops
```
Comment (khesoem) [inline contributions/presentation/week3/gchang-wska/README.md]: Can you elaborate on this? Do you mean what tools are used? Do you mean you want to discuss how developers decide what coverage metrics they should use? Or ...?

File: contributions/presentation/week3/gchang-wska/README.md [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+
+## Members
+Gabriel Chang (gchang@kth.se)
+Github username: ChangGabriel
+
+William Skagerström (wska@kth.se)
+Github username: wska
+
+## Topic
+An introduction to code coverage
```
Comment (khesoem) [inline contributions/presentation/week3/gchang-wska/README.md]: Code coverage is a broad topic and an introduction to code coverage could be a whole course. Instead, you can focus on a more narrow topic, such as "code coverage in continuous integration".

**Comment by khesoem** [comment]
> Is week 3 from 05 april - 09 april or 12 april-16 april?

Week 3 session will be held on "[Tue 13 apr](https://www.kth.se/social/course/DD2482/calendar/)".

**Comment by ChangGabriel** [comment]
Hi, thanks for input. We have now updated the proposal to be more detailed.

**Comment by khesoem** [comment]
@ChangGabriel I just found a couple of PRs from last year that are similar to your work. Including this one:
https://github.com/KTH/devops-course/pull/450

To be unique, I think you need to make your proposal even more specific. **Why don't you focus on a specific tool?**

**Comment by khesoem** [comment]
Since this is a duplicated proposal, I have to close it. We'll review your new proposal as soon as we receive it.

**Pull request closed without merging**